src/test/java/gov/cabinetoffice/gapfindapiadmin/controllers/security/JwtAuthorisationFilterTest.java

```diff
@@ -0,0 +1,121 @@
+package gov.cabinetoffice.gapfindapiadmin.controllers.security;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import gov.cabinetoffice.gapfindapiadmin.config.OneLoginConfig;
+import gov.cabinetoffice.gapfindapiadmin.exceptions.UnauthorizedException;
+import gov.cabinetoffice.gapfindapiadmin.models.JwtPayload;
+import gov.cabinetoffice.gapfindapiadmin.security.JwtAuthorisationFilter;
+import gov.cabinetoffice.gapfindapiadmin.services.JwtService;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+import static org.mockito.Mockito.*;
+
+
+@ExtendWith(MockitoExtension.class)
+public class JwtAuthorisationFilterTest {
+
+    private JwtAuthorisationFilter jwtAuthorisationFilter;
+
+    @Mock
+    private JwtService jwtService;
+
+    @Mock
+    private OneLoginConfig oneLoginConfig;
+
+    @Mock
+    private HttpServletRequest request;
+
+    @Mock
+    private HttpServletResponse response;
+
+    @Mock
+    private FilterChain filterChain;
+
+    private final String jwt = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJmdW5kZXJfaWQiOiIxIn0.NbYwfVIpqalW1gM204pQXM7o6voNoO7EyFwl1XjXEQQ";
```

Comment:
I don't love this approach to testing JWTs and I think we should change it later but considering we do this everywhere else it should be fine for now.